### PR TITLE
OCPEDGE-805: feat: simplifying the single node + workers jobs

### PR DESCRIPTION
**Pull request by jeff-roche**
Removing some redundancy in the sno+workers jobs and simplifying the workflow to be similar to the standard SNO jobs

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml [hunk 1]
```diff
@@ -81,21 +81,27 @@ resources:
...
-  timeout: 3h0m0s
-- as: openshift-e2e-aws-single-node-workers-upgrade-conformance
-  cron: 58 4 8,23 * *
+- as: e2e-aws-ovn-single-node-workers
+  interval: 168h
   steps:
     cluster_profile: aws-2
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
-    workflow: openshift-e2e-aws-single-node-workers-upgrade-conformance
```
Comment (jakobmoellerdev) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml:98]: is the upgrade conformance now implicit?

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml [hunk 2]
```diff
@@ -81,21 +81,27 @@ resources:
...
+- as: e2e-aws-ovn-single-node-workers-serial
+  interval: 168h
   steps:
     cluster_profile: aws-2
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
+      TEST_SUITE: openshift/conformance/serial
+    observers:
+      enable:
+      - observers-resource-watch
```
Comment (jakobmoellerdev) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml:93]: why is that necessary?

**Review by jakobmoellerdev — COMMENTED** [review]
I think this is mostly LGTM , but there is a big maybe unintended changeset in here. Maybe needs a rebase

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml (hunk 1 above)
Comment (jeff-roche) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml:98]: Yeah same way the openshift-e2e-aws-single-node has conformance as implicit

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml (hunk 2 above)
Comment (jeff-roche) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml:93]: This adds the cluster resources observer that lets us take a look at performance impacts. We do the same thing for the regular single node lanes so I figured it'd be a good idea to monitor it here as well

**Review by jakobmoellerdev — COMMENTED** [review]
/lgtm

**Review by vrutkovs — APPROVED** [review]
/approve

**Comment by jeff-roche** [comment]
/pj-rehearse ack

Nothing changed outside of a rebase so no need to re-run rehearsals
